llvm/lib/Target/AArch64/AsmParser/AArch64AsmParser.cpp

```diff
@@ -6714,7 +6750,7 @@ bool AArch64AsmParser::MatchAndEmitInstruction(SMLoc IDLoc, unsigned &Opcode,
   case Match_InvalidSVEVectorListStrided4x16:
   case Match_InvalidSVEVectorListStrided4x32:
   case Match_InvalidSVEVectorListStrided4x64:
-  case Match_InvalidSVEPNRasPPRPredicateBReg:
+  case Match_InvalidSVEPPRorPNRBReg:
```

Comment:
nit: should this be together with `case Match_InvalidSVEPPRorPNRAnyReg`?

Reply:
If we keep it here we retain the same error messages as the older reg class. I can move it up but it will require updating quite a few tests.